### ISSUE TITLE
Wrong predictive back order when opening video from player

### ISSUE THREAD

**Issue by FineFindus** [libre-tube/LibreTube#6880]
### Steps to reproduce

1. Open a video
2. Go to the channel
3. Play another video
4. Try to use predictive back

### Expected behavior

The back gesture should first close the player and then the channel page.

### Actual behavior

Currently, the channel page is closed first, behind the player. Only after closing it the player can be closed.

### LibreTube version

master

### Android version

Android 15 QPR 1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
